app/browser/reducers/historyReducer.js

```diff
@@ -4,6 +4,7 @@
 
 const Immutable = require('immutable')
 const BrowserWindow = require('electron').BrowserWindow
+const {URL} = require('url')
```

Comment:
please use `app/common/urlParse.js` instead

Reply:
Will do! I'll also use `useParse` here to keep it consistent

https://github.com/brave/browser-laptop/pull/12754/files#diff-b9f459fd47114771333b78b87fac7daaR363